ksp_plugin_adapter/reference_frame_selector.cs

```diff
@@ -170,38 +181,34 @@ public enum FrameType {
                                    CelestialBody selected,
                                    Vessel target_override) {
     if (target_override) {
-      return "Reference frame fixing the target vessel (" +
-             target_override.vesselName + "), the plane of its orbit around " +
-             selected.NameWithArticle() + ", and the line between them";
+      return Localizer.Format("#Principia_RFS_DynamicText_Target",
```

Comment:
Principia_ReferenceFrameSelector_Description_Target, and same below in this function, Principia_ReferenceFrameSelector_Description_BodyCentredNonRotating etc.